src/api/app/policies/project_policy.rb

```diff
@@ -49,7 +49,7 @@ def accept?
     record.staged_requests.each do |request|
       # we pretend the user asked for force, we only want to check permissions
       # not if it makes sense
-      request.permission_check_change_state!(newstate: 'accepted', force: true)
+      return false unless request.permission_check_change_state(newstate: 'accepted', force: true)
```

Comment:
note that force accept is no longer an option if you're not target maintainer

Reply:
As far as I can see this is just about https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request_permission_check.rb#L354-L355

Reply:
Jein. The policy uses force and skips the review check by that - but we later check that the project is in acceptable and so staging projects in review get blocked on a different level. The task of the policy is to check if it's technically possible, not if the staging project is "acceptable" or not.

But we also checked that the accepting user is target maintainer and so we *could* skip review checks. If this is no longer guaranteed, we need to check that the request is actually acceptable in full glory. I.e. if accepting the last review *really* will accept the request by approval.

Reply:
> The task of the policy is to check if it's technically possible, not if the staging project is "acceptable" or not.

The policy is about authorization, checking permissions. If someone is allowed to do something. But as you know things are tangled deeply...

>  if accepting the last review really will accept the request by approval

That is what we do no?